base/sparse/sparsematrix.jl

```diff
@@ -458,18 +520,57 @@ function sprand{T}(m::Integer, n::Integer, density::AbstractFloat,
     sparse_IJ_sorted!(I, J, rfn(length(I)), m, n, AddFun())  # it will never need to combine
 end
 
+"""
+```rst
+..  sprand([rng,] m,n,p [,rfn])
+
+Create a random ``m`` by ``n`` sparse matrix, in which the probability of any
+element being nonzero is independently given by ``p`` (and hence the mean
+density of nonzeros is also exactly ``p``). Nonzero values are sampled from
+the distribution specified by ``rfn``. The uniform distribution is used in
+case ``rfn`` is not specified. The optional ``rng`` argument specifies a
+random number generator, see :ref:`Random Numbers <random-numbers>`.
+```
+"""
```

Comment:
would it make more sense to put this on the method that has both `rng` and `rfn` arguments?